app/frontend/src/lib/components/LogRow.svelte

```diff
@@ -0,0 +1,89 @@
+<script lang="ts">
+  import { formatSpeed } from "$lib/utilities"
+
+  import { Plus, Minus, Dot } from "@lucide/svelte"
+
+  let { log } = $props()
+
+  const iconByState = {
+    added: Plus,
+    removed: Minus,
+    normal: Dot,
+  } as const
+
+  const LogIcon = iconByState[log.State as keyof typeof iconByState] ?? Dot
+</script>
+
+<code class={`log-row ${log.State}`}>
+  <div class="primary">
```

Comment:
```suggestion
  <div class="left">
```
Might just be clearer of a name?

Reply:
I agree, change made.